src/Npgsql/NpgsqlCommand.cs

```diff
@@ -1142,7 +1142,7 @@ public new Task<NpgsqlDataReader> ExecuteReaderAsync(CommandBehavior behavior, C
         internal ManualResetValueTaskSource<NpgsqlConnector> ExecutionCompletion { get; }
             = new ManualResetValueTaskSource<NpgsqlConnector>();
 
-        async ValueTask<NpgsqlDataReader> ExecuteReaderAsync(CommandBehavior behavior, bool async, CancellationToken cancellationToken)
+        internal async ValueTask<NpgsqlDataReader> ExecuteReaderAsync(CommandBehavior behavior, bool async, CancellationToken cancellationToken)
```

Comment:
As written below, we shouldn't call these directly unless there's a good reason - better to call the public ones.

Reply:
Agreed!